lib/rucio/rse/__init__.py

```diff
@@ -46,12 +46,23 @@ def get_rse_client(rse, **kwarg):
     return RSEClient().get_rse(rse)
 
 
-def get_signed_url_client(service, op, url):
+def get_signed_url_client(rse, service, op, url):
```

Comment:
this should also be `rse_id`, right?

Reply:
In this case it is an RSE name rather than an ID. The API layer translates it to an ID and passes that into the core.